### PR TITLE
HCX-53 Add method for adding document-subject bind

### PR DESCRIPTION
**Pull request by dolfinus**

### CODE REVIEW

File: src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy [hunk 1]
```diff
@@ -486,6 +486,48 @@ trait Document {
...
+   * Create document-subject bind
+   *
+   * Overload with positional args
+   * @see #addDocumentSubject(Map, def)
+   */
+  Boolean addDocumentSubject(Map input = [:], def docId, def subjectId, def accountId = null) {
+    return putDocumentSubject(input + [docId: docId, subjectId: subjectId, accountId: accountId])
+  }
+
+    /**
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy]: unnecessary space

File: src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy [hunk 2]
```diff
@@ -486,6 +486,48 @@ trait Document {
...
+   */
+  Boolean addDocumentProvider(Map input = [:], def docId, def subjectId, def accountId = null) {
+    return addDocumentSubject(input + [roleId: getProviderRoleId()], docId, subjectId, accountId)
+  }
+
+  /**
+   * Create document-subject receiver
+   * @see #addDocumentSubject(Map, def, def)
+   */
+  Boolean addDocumentReceiver(Map input = [:], def docId, def subjectId, def accountId = null) {
```
Comment (sugarfree1) [inline src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy]: we call it recipient now

File: src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy (hunk 1 above)
Comment (dolfinus) [reply src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy]: Which one?

File: src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy (hunk 2 above)
Comment (dolfinus) [reply src/org/camunda/latera/bss/connectors/hid/hydra/Document.groovy]: ok

**Pull request merged**